### ISSUE TITLE
Is it possible to train multiple subjects; for example a style and a person?

### ISSUE THREAD

**Issue by Barry-Bot1** [TheLastBen/fast-stable-diffusion#686]
Would it just be renaming the images to the corresponding subjects?

**Comment by TheLastBen** [comment]
Yes, it is possible, just avoid training two subject from the same type like two men or two women, they will get merged, try a man and a woman, a style (without faces), an object, and animal, all at the same time and it will work
